server/routes/licenceRoutes.ts

```diff
@@ -68,4 +68,137 @@ export default class LicenceRoutes {
       { filename, pdfOptions: { ...pdfOptions, headerHtml, footerHtml } }
     )
   }
+
+  public getCaseloadForLicenceCreation: RequestHandler = async (req, res): Promise<void> => {
+    const caseload = [
+      {
+        name: 'Adam Balasaravika',
+        crnNumber: 'X381306',
+        conditionalReleaseDate: '03 August 2022',
+      },
+    ]
+    res.render('pages/create/caseload', { caseload })
+  }
+
+  public getInitialMeetingPerson: RequestHandler = async (req, res): Promise<void> => {
+    const offender = {
+      name: 'Adam Balasaravika',
+    }
+    res.render('pages/create/initialMeetingPerson', { offender })
+  }
+
+  public postInitialMeetingPerson: RequestHandler = async (req, res): Promise<void> => {
+    const id = 1
+    res.redirect(`/licence/create/id/${id}/initial-meeting-place`)
+  }
+
+  public getInitialMeetingPlace: RequestHandler = async (req, res): Promise<void> => {
+    const offender = {
+      name: 'Adam Balasaravika',
+    }
+    res.render('pages/create/initialMeetingPlace', { offender })
+  }
+
+  public postInitialMeetingPlace: RequestHandler = async (req, res): Promise<void> => {
+    const { id } = req.params
+    res.redirect(`/licence/create/id/${id}/initial-meeting-contact`)
+  }
+
+  public getInitialMeetingContact: RequestHandler = async (req, res): Promise<void> => {
+    res.render('pages/create/initialMeetingContact', {})
+  }
+
+  public postInitialMeetingContact: RequestHandler = async (req, res): Promise<void> => {
+    const { id } = req.params
+    res.redirect(`/licence/create/id/${id}/initial-meeting-time`)
+  }
+
+  public getInitialMeetingTime: RequestHandler = async (req, res): Promise<void> => {
+    const offender = {
+      name: 'Adam Balasaravika',
+    }
```

Comment:
Good for now, and we can extend this to call the licence service to supply the data rather than stub in the routes.